### PR TITLE
refactor(e2e): Replace latency emulation Python script by generating a shell script in e2e's setup

### PR DESCRIPTION
**Pull request by hvanz**
Solves #2666 

Currently latency emulation is set up on each e2e node by running a Python script that requires two csv files, one with information about all the nodes' IP addresses and another with latency values between different data-center zones.

This PR generates, for each node in the testnet, a single shell script with the same `tc` (traffic control) commands as the Python scripts. The shell script is then called on the entry point of the Docker image.

To reviewers: follow the commits.

---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments

### CODE REVIEW

**Review by melekes — APPROVED** [review]
👍

File: test/e2e/runner/setup.go [hunk 1]
```diff
@@ -500,3 +516,78 @@ func UpdateConfigStateSync(node *e2e.Node, height int64, hash []byte) error {
 	bz = regexp.MustCompile(`(?m)^trust_hash =.*`).ReplaceAll(bz, []byte(fmt.Sprintf(`trust_hash = "%X"`, hash)))
 	return os.WriteFile(cfgPath, bz, 0o644) //nolint:gosec
 }
+
+// tcCommands generates a list of tc (traffic control) commands to emulate
```
Comment (cason) [inline test/e2e/runner/setup.go:520]: So this guy creates the `emulate-latency.sh` script?

File: test/e2e/docker/entrypoint
Comment (cason) [inline test/e2e/docker/entrypoint:1]: Shouldn't we change this on other entrypoints? E.g., the `builtin` one?

File: test/e2e/runner/setup.go (hunk 1 above)
Comment (cason) [reply test/e2e/runner/setup.go:520]: Do you mind moving this function to a different file, like `latency.go` or `latency_emulation.go`?

**Review by cason — APPROVED** [review]
Thank for that, very helpful removing `python` requirement and script from the image :)

File: test/e2e/runner/setup.go (hunk 1 above)
Comment (hvanz) [reply test/e2e/runner/setup.go]: `tcCommands` generates the content of `emulate-latency.sh`. I moved this function to to a new file `latency_emulation.go`.

**Pull request merged**